### ISSUE TITLE
Header Text Looks Shitty?

### ISSUE THREAD

**Issue by StrangeWill** [devanooga/devanooga.com#158]
![image](https://user-images.githubusercontent.com/1322565/29646602-7da409fc-8852-11e7-949c-1667171bbd8a.png)

It looks like it's not anti-aliased or something, is that just me? Derp?

**Comment by DigTK** [comment]
Nope, I see it too

![image](https://user-images.githubusercontent.com/4950390/29646656-c818d85a-8852-11e7-8a8b-dfdb0871c343.png)

**Comment by agarzola** [comment]
That’s odd. I’ll take a look at it this weekend.

**Comment by DigTK** [comment]
It's not just the header, by the way. Is it the fault of the font?
![image](https://user-images.githubusercontent.com/4950390/30138475-8e4ada80-9336-11e7-93f8-b5d3935161a2.png)

**Comment by DigTK** [comment]
This looks like a solution to our issue: https://stackoverflow.com/questions/11487427/is-there-any-font-smoothing-in-google-chrome/19582105#19582105

but I don't know enough about web-development to properly implement this

**Comment by agarzola** [comment]
In order to implement that, we’ll have to download the font files from Google Fonts and serve them from our own hosting solution. That’s not a bad idea, considering it makes us less reliant on external services, but it might have minor performance implications.

Actionable items:
- **Download all formats available for the Asap typeface.**
  The weights we need are: 400, 400 Italic, and 500; the subsets we need are `latin` and `latin-ext`. They should be added to a `fonts/` directory at the root of the project.
- **Remove `_includes/google-fonts.html` file.**
- **Create CSS in a `_sass/_google-fonts.scss` file.**
  It should blend the technique detailed in that article with [the CSS delivered by Google](https://fonts.googleapis.com/css?family=Asap:400,400i,500&amp;subset=latin-ext).
- **Test the shit out of it.**

**Comment by DigTK** [comment]
I actually fiddled around with that during last hacknight, you can get the fonts easily here:

https://google-webfonts-helper.herokuapp.com/fonts/asap?subsets=latin,latin-ext

It even prepares the css for you, but css is my arch-nemesis and for some reason I couldn't get it to work.

**Comment by StrangeWill** [comment]
@agarzola do you have a similar font you could recommend?

**Comment by StrangeWill** [comment]
You can reproduce on this here: https://google-webfonts-helper.herokuapp.com/fonts/asap?subsets=latin

I could try basically overriding the woff files provided in the github issue maybe? Not sure if I can partially replace a font file easily.

**Comment by StrangeWill** [comment]
Yeah the files that come in that zip is all there is, so if I need more files I'm worried about those even being available.

May just shelve this as a Google problem and wait on them fixing it then...

**Comment by StrangeWill** [comment]
@thetest3r recommend we look at https://fonts.google.com/specimen/Raleway and it looks similar.
